src/NuGet.Clients/VsExtension/NuGetPackage.cs

```diff
@@ -103,6 +105,13 @@ public sealed class NuGetPackage : Package, IVsPackageExtensionProvider, IVsPers
 
         public NuGetPackage()
         {
+            // Set IsDev14 in NuGet.Common.RunTimeHelper to pick %PROGRAMDATA%/NuGet/Config as the MachineWideBaseDir
+            // This is only needed for Dev14 VSIX
+            var dte = ServiceLocator.GetInstance<DTE>();
+            if (VSVersionHelper.IsDev14(dte))
+            {
+                RuntimeEnvironmentHelper.IsDev14 = true;
+            }
```

Comment:
I don't think this needs to be a runtime check.

    #if VS14
        RuntimeEnvironmentHelper.IsDev14 = true;
    #endif